sdk/containerregistry/azure-containerregistry/azure/containerregistry/_enums.py

```diff
@@ -0,0 +1,17 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+# pylint:skip-file (avoids crash due to six.with_metaclass https://github.com/PyCQA/astroid/issues/713)
+from enum import Enum
+from six import with_metaclass
+
+from azure.core import CaseInsensitiveEnumMeta
+
+class ContainerRegistryAudience(with_metaclass(CaseInsensitiveEnumMeta, str, Enum)):
+    """Supported container registry audience"""
+
+    ARM_CHINA_VALUE = "https://management.chinacloudapi.cn"
```

Comment:
In .NET we've called the enum values `AzureResourceManagerChina`, e.g., because we have ARM in another enum indicating processor architecture, and we don't have the `_VALUE` suffix.  See [the APIView here](https://apiview.dev/Assemblies/Review/7ebfcb36ff6149ba96170bd059094513#Azure.Containers.ContainerRegistry.ContainerRegistryAudience).  I don't have a strong opinion regarding Python values, but I'd suggest to follow the precedent set elsewhere in Python APIs.

Reply:
In [azure-identity](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/azure/identity/_constants.py#L14) we have `AZURE_CHINA`, `AZURE_GERMANY`, etc. so it makes sense to drop `_VALUE`. I think using `ARM_` instead of `AZURE_RESOURCE_MANAGER_` would be nice for brevity, but that's a question for Anna and Johan

Reply:
@johanste - having looked into the cloud configuration topic - do you have any concerns with `ARM_CHINA/ARM_GERMANY/ARM_FOO` naming convention?

Reply:
I would just document the possible values. No need for a public type.

Reply:
(which also side-steps the naming convention question :))